routes/conversation.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const router = express.Router();
+const Invitation = require('../models/Invitation');
+const Conversation = require('../models/Conversation');
+const User = require('../models/User');
+const auth = require('../middleware/auth');
+
+router.post('/:members', auth, async (req, res) => {
+  const memberEmails = req.params.members.split('&');
+  const membersFromDb = [];
+
+  try {
+    for (const email of memberEmails) {
+      const foundMember = await User.findOne({ email });
+      if (foundMember) membersFromDb.push(foundMember);
+    }
+
+    const conversation = new Conversation({
+      members: membersFromDb,
+    });
+
+    await conversation.save();
```

Comment:
How will conversations be updated? Should we return the new conversation id to use in future put requests?

Reply:
If I don't end up deleting this I think we would want to res.status(200).json(conversation);